comments-on-conformance.md

```diff
@@ -7,10 +7,10 @@ WCAG2ICT is not a standard, so it is not possible to conform to WCAG2ICT. Howeve
 3.  WCAG 2 conformance is applied to the item being evaluated (i.e. web page) as a whole, except when a process includes use of several items, in which case all of the items that are needed in order to complete the process must conform.
 4.  In WCAG 2, when conformance depends upon accessibility features of the platform (i.e. browser for web content) or on assistive technologies, WCAG 2 requires that there are assistive technologies, etc. that work with the product (web page).  That is, conformance with WCAG 2 requires that [only accessibility-supported ways of using technologies](https://www.w3.org/TR/WCAG22/#cc4) are relied upon to satisfy the success criteria.  WCAG 2 requires that any information or functionality that is provided in a way that is not accessibility supported is also available in a way that is accessibility supported.
-5.  WCAG 2 allows information on part of a page to not conform if the same information is available elsewhere on the page in conforming fashion. However WCAG 2 identifies 4 success criteria that must be met on all areas of the page because they can interfere with the user's ability to access and use other parts of the page:
+5.  WCAG 2 allows information on part of a page to not conform if the same information is available elsewhere on the page in a conforming fashion. However WCAG 2 identifies four success criteria that must be met on all areas of the page because they can interfere with the user's ability to access and use other parts of the page:
     *   [1.4.2 Audio Control](http://www.w3.org/TR/WCAG22/#audio-control);
     *   [2.1.2 No Keyboard Trap](http://www.w3.org/TR/WCAG22/#no-keyboard-trap);
     *   [2.2.2 Pause, Stop, Hide](http://www.w3.org/TR/WCAG22/#pause-stop-hide).
     *   [2.3.1 Three Flashes or Below Threshold](http://www.w3.org/TR/WCAG22/#three-flashes-or-below-threshold);
 
-Also, as noted in the Introduction, it wasn't possible to unambiguously carve up software into discrete pieces, and so the unit of evaluation for non-web software is the whole software program. As with any software testing this can be a very large unit of evaluation, and methods similar to standard software testing might be used.
+Also, as noted in the Introduction, it wasn't possible to unambiguously carve up software into discrete pieces, and so the unit of evaluation for non-web software is the whole software program.  As with any software testing, this can be a very large unit of evaluation, and methods similar to standard software testing may be needed.
```

Comment:
```suggestion
Also, as noted in the Introduction, it wasn't possible to unambiguously carve up software into discrete pieces, and so the unit of evaluation for non-web software is the whole software program. As with any software testing, this can be a very large unit of evaluation, and methods similar to standard software testing may be needed.
```

Reply:
remove extra space